js/render/draw_circle.js

```diff
@@ -4,6 +4,30 @@ var browser = require('../util/browser.js');
 
 module.exports = drawCircles;
 
+var PROPERTIES = [
+    {
+        styleName: 'circle-color',
+        styleType: 'color',
+        glName: 'a_color',
+        glWidth: 4,
+        glType: '4fv'
+    },
+    {
+        styleName: 'circle-blur',
+        styleType: 'number',
+        glName: 'a_blur',
+        glWidth: 2,
+        glType: '1f'
+    },
+    {
+        styleName: 'circle-radius',
+        styleType: 'number',
+        glName: 'a_size',
+        glWidth: 2,
+        glType: '1f'
+    }
+];
+
```

Comment:
This was a small trial of a larger refactoring I want to do of the buffer generation code. Building better abstractions for buffer generation will make our code shorter and therefore less buggy and easier to maintain. Tracking the larger issue in #1318 (though there aren't any more details there right now).